libpod/runtime_ctr.go

```diff
@@ -836,3 +836,49 @@ func (r *Runtime) GetLatestContainer() (*Container, error) {
 	}
 	return ctrs[lastCreatedIndex], nil
 }
+
+// PruneContainers removes stopped and exited containers from localstorage.  A set of optional filters
+// can be provided to be more granular.
+func (r *Runtime) PruneContainers(filterFuncs []ContainerFilter) ([]define.PruneContainerResponse, error) {
+	var (
+		prunedResponses []define.PruneContainerResponse
+	)
+	// We add getting the exited and stopped containers via a filter
+	containerStateFilter := func(c *Container) bool {
+		state, err := c.State()
+		if err != nil {
+			logrus.Error(err)
+			return false
+		}
+		if c.PodID() != "" {
+			return false
+		}
+		if state == define.ContainerStateStopped || state == define.ContainerStateExited {
+			return true
+		}
+		return false
+	}
+	filterFuncs = append(filterFuncs, containerStateFilter)
+
+	delContainers, err := r.GetContainers(filterFuncs...)
+	if err != nil {
+		return nil, err
+	}
+	for _, c := range delContainers {
+		ctr := c
+		response := define.PruneContainerResponse{
+			ID: ctr.ID(),
+		}
+		size, err := ctr.RootFsSize()
```

Comment:
Shouldn't this be RwSize() - we're not removing the image the container is based on, so I don't think we freed that space.